docs/01-programs/02-candy-machine/10-how-to-guides/00-my-first-candy-machine-part1.md

```diff
@@ -0,0 +1,658 @@
+  "awsConfig": null,
+  "nftStorageAuthToken": null,
+  "shdwStorageAccount": null,
+  "pinataConfig": null,
+  "hiddenSettings": null,
+  "guards": {
+    "default": {
+      "mintLimit": {
+        "id": 1,
+        "limit": 3
+      },
+      "solPayment": {
+        "value": 0.5,
+        "destination": "Tes1zkZkXhgTaMFqVgbgvMsVkRJpq4Y6g54SbDBeKVV"
+      },
+      "startDate": {
+        "date": "2022-10-23T20:00:00Z"
+      }
+    }
+  }
+}
+```
+
+</AccordionCode>
+</Accordion>
+
+We added a default guard with the following settings:
+- `mintLimit` only allows minting a maximum of three NFTs (`limit`) per wallet.
+- `solPayment` requires a payment of 0.5 SOL (`value`) which is paid to the destination wallet (`destination`).
+- `startDate` restricts the mint function to not allow minting before `2022-10-23T20:00:00Z` (`date`)
+
+You can add multiple different guards e.g. to create multiple phases for OGs, WL and public mint. You can find more information about the possibilities on the [Guards](/programs/candy-machine/candy-guards) pages.
+
+After modifying and saving the manually modified config file we deploy the guard and set it as the guard of our candy machine. Afterwards, it will only be possible to mint through that guard if all the restrictions are matched. You'll have to use the following command for it:
+
+<Accordion>
+<AccordionCode title="Command"  open={true}>
+
+```
+sugar guard add
+```
+
+</AccordionCode>
+<AccordionCode title="Output">
+
+```bash
+[1/3] 🔍 Looking up candy machine
+
+Candy machine ID: Ews3L5NoAjjLEHYqEu47DqQ77nsqgNQs3NuELjBCd5bb
+
+[2/3] 🖥  Loading candy guard
+▪▪▪▪▪ Done
+
+Candy guard ID: 7XJdGaywrtcEpohrcZ7kYeMjEHNFVq1XjVpBQ6Doi7TP
+
+[3/3] 📦 Wrapping
+Signature: 3C7iXdu9msYviFAJz4esMvkMAiSXTfExBYr9E5VEJfrmJnojrfoEEDPGzJ7qGDx1J5Pf6bHZtfRqYvBPLrc35on5
+
+The candy guard is now the mint authority of the candy machine.
+
+✅ Command successful.
+```
+
+</AccordionCode>
+</Accordion>
+
+If required you can modify the guard afterwards by running:
+
+<Accordion>
+<AccordionCode title="Command"  open={true}>
+
+```bash
+sugar guard update
+```
+
+</AccordionCode>
+<AccordionCode title="Output">
+
+```bash
+[1/2] 🔍 Loading candy guard
+▪▪▪▪▪ Done
+Candy guard ID: 7XJdGaywrtcEpohrcZ7kYeMjEHNFVq1XjVpBQ6Doi7TP
+
+[2/2] 🖥  Updating configuration
+Signature: 43oDgKfXeFx3erZbNihJ1a5KxcUPdDLcoB2SW6ZytaxjBhE1M6quSWUyj3kdG64h4aXTZqUj6NiQULbZqpVJABbw
+
+✅ Command successful.
+```
+
+</AccordionCode>
+</Accordion>
+
+Finally, to see the current guard settings, you may run to following command:
+
+<Accordion>
+<AccordionCode title="Command"  open={true}>
+
+```bash
+sugar guard show
+```
+
+</AccordionCode>
+<AccordionCode title="Output">
+
+```bash
+[1/2] 🔍 Loading candy guard
+▪▪▪▪▪ Done
+Candy guard ID: 7XJdGaywrtcEpohrcZ7kYeMjEHNFVq1XjVpBQ6Doi7TP
+
+[2/2] 🖥  Updating configuration
+Signature: 5nQ1dasS3QAQJH7fBwFXd2VgyRirfYmnZ8XKgsfvWubX4hJ6jzY4dzqEVpXpULgw1PsC5eyJriSMCnVXxw2Mckae
+
+✅ Command successful.
```

Comment:
I'm a bit confused because the `sugar guard show` command doesn't actually show any guard settings. Is that output correct?

Reply:
you are right, this is a copy paste error. Fixed it.